server/basicServices/basicServices/BasicServicesService.js

```diff
@@ -1444,6 +1449,14 @@ exports.updateOperationKey = function (body, user, originator, xCorrelator, trac
   });
 }
 
+async function isForwardingNameExist(forwardingName) {
+  const forwardingConstruct = await ForwardingDomain.getForwardingConstructForTheForwardingNameAsync(forwardingName);
+  if (forwardingConstruct === undefined) {
```

Comment:
return forwardingConstruct !== undefined;

Reply:
Thanks for proposing the optimization.